build/source/dshare/globalData.f90

```diff
@@ -170,9 +170,11 @@ MODULE globalData
  integer(i4b),save,public                    :: numtim                  ! number of time steps
  real(dp),save,public                        :: data_step               ! time step of the data
  real(dp),save,public                        :: refJulday               ! reference time in fractional julian days
+ real(dp),save,public                        :: refJulday_data          ! reference time in fractional julian days (data files)
```

Comment:
What's the difference between with `refJulday` and `refJulday_data`?

Reply:
`refJulday` is the reference time for the model simulation and `refJulday_data` is the reference time in the data files (as noted in the comments). These can be different.